.nucleus.yaml

```diff
@@ -1,26 +1,61 @@
+# shared config. The `__shared_config__` name isn't meaningful.
+__shared_config__: &branch-definition
```

Comment:
Nucleus doesn't look happy about `__shared_config__` being set at the top level.

Reply:
I'll use another technique for anchors.